pkg/registry/pod/strategy.go

```diff
@@ -159,13 +159,14 @@ func MatchPod(label labels.Selector, field fields.Selector) generic.Matcher {
 	}
 }
 
-// PodToSelectableFields returns a label set that represents the object
+// PodToSelectableFields returns a field set that represents the object
 // TODO: fields are not labels, and the validation rules for them do not apply.
 func PodToSelectableFields(pod *api.Pod) fields.Set {
 	return fields.Set{
-		"metadata.name": pod.Name,
-		"spec.nodeName": pod.Spec.NodeName,
-		"status.phase":  string(pod.Status.Phase),
+		"metadata.name":      pod.Name,
```

Comment:
It would be good if we had a utility that could take an ObjectMeta and return ObjectMeta selectable fields that is merged with each objects field specific set.  In that way, DaemonSet, Secret, etc. could all use the same core utility and get a common set of selectors.